src/site/serverless/affordable-housing-table.liquid

```diff
@@ -18,7 +18,9 @@ pageHead: '<style type="text/css" media="print">@page {size: landscape;}</style>
 {% endif %}
 
 <div class="flex align_end">
-  <h3>Showing {{numResults}} {% if numResults == 1 %}property{% else %}properties{% endif %}</h3>
+   <!-- <h3>Showing {{numResults}} {% if numResults == 1 %}property{% else %}properties{% endif %}</h3> -->
```

Comment:
You should delete this code rather than commenting it out.  The entire revision history is saved, so we can always go back and see how the code was prior to your change.

You should be able to delete this line, commit, and push to `my-test-fob` without updating this pull request.  It will update automatically.